utility_network/display-utility-associations/src/main/resources/style.css

```diff
@@ -0,0 +1,13 @@
+.panel-region .label {
+  -fx-text-fill: white;
+}
+
+.label {
+  -fx-text-fill: white;
+  -fx-font-size: 12px;
+  -fx-wrap-text: true;
+}
+
+.panel-region .radio-button {
+   -fx-text-fill: white;
+}
```

Comment:
We can get rid of these two rules, since the only labels we have are in `panel-region`, and we don't have a radio button at all
```suggestion
```

Reply:
Another note, we recently had to move the CSS file to a different folder for these rules to catch properly in the Sample Viewer:

`utility_network/display-utility-associations/src/main/resources/display_utility_associations/style.css `